experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -898,6 +941,140 @@ describe('Node SDK', () => {
       await sdk.shutdown();
     });
   });
+
+  describe('configuring logger provider from env', () => {
```

Comment:
Perhaps add a test that an OTLP http/protobuf logs exporter is setup by default. I don't think there is a test case for that.